src/main/webapp/sign-up.html

```diff
@@ -23,5 +23,14 @@ <h1>Sign Up</h1>
         <br />
         <input type="submit" />
     </form>
+
```

Comment:
I think for the above form to send data to your servlet, you will want to update the action to be /sign-up with a method of POST.